src/components/Transcriber.tsx

```diff
@@ -1303,7 +1304,13 @@ export function Transcriber(
         {confirm && (
           <Confirm
             isDelete={false}
-            text={t.updateByOther2.replace('{0}', confirm.transcription)}
+            text={t.updateByOther2
+              .replace(
+                '{0}',
+                GetUser(memory, related(mediafile, 'lastModifiedByUser'))
+                  .attributes?.name ?? 'unknown'
+              )
+              .replace('{1}', confirm.transcription)}
```

Comment:
There isn't an issue for this.  I threw it in to avoid changing strings files multiple times (which means you're right and it will be hard to extricate it even now).  We aren't going to be cherry picking this because this is a convenience improvement but no new functionality.